packages/build/tests/core/tests.mjs

```diff
@@ -306,7 +303,9 @@ test('Print warning when headers file is missing from publish directory', async
 })
 
 test.serial('Successfully builds ES module function with feature flag', async (t) => {
+  // eslint-disable-next-line import/no-named-as-default-member
```

Comment:
This seems to be an issue with Sinon itself, or with ESLint. The import of `sinon` is in fact correct.